python/mlc_chat/cli/delivery.py

```diff
@@ -29,9 +36,15 @@ class ModelInfo:
     model_id: str
     model: Path
     conv_template: str
-    context_window_size: int
     quantization: str
     source_format: str = "auto"
+    # If unspecified in CLI, remains to be None and will not be
+    # passed to `gen_config` or `convert_weight`
+    context_window_size: int = None
+    sliding_window_size: int = None
+    prefill_chunk_size: int = None
+    attention_sink_size: int = None
+    tensor_parallel_shards: int = None
```

Comment:
How about having some default values for these two? How about:
```suggestion
    attention_sink_size: int = 4
    tensor_parallel_shards: int = 1
```

Reply:
I was thinking that since `ModelInfo` is used to generate the command `mlc_chat gen_config`, we might want to make it consistent to the arguments in [python/mlc_chat/cli/gen_config.py](https://github.com/mlc-ai/mlc-llm/blob/main/python/mlc_chat/cli/gen_config.py), where the default is `None`. Values like `4` for attention sink would still show up even if `delivery.py` does not specify it.

Reply:
Got it. That makes sense!